roadmap.md

```diff
@@ -1,60 +1,41 @@
 # Roadmap
 
-The MSAL.js team schedules work on a quarterly basis. The roadmap below provides a subset of the team's quarterly commitments for past and upcoming quarters. 
+The MSAL.js team's roadmap is presented as near-term, mid-term, & long-term to give a people a better idea of what we are currently working on, what is up next for us, and what we are considering for the future. These are not guarenteed timelines, but insight into the team's current direction. The roadmap is focused on big features or focus areas; it may not call out each individual work item that is added to the library. 
 
-## Glossary
+If you have feedback or suggestions on our roadmap, please let us know.
 
-- **CY**: Calendar year, January to December.
-  - Example: "CY2021" refers to January 2021 through December 2021.
-- **FY**: Microsoft fiscal Year, July to June.
-  - Example: "FY2022" refers to July 2021 through June 2022. 
-- **Q**: Quarter, three month period.
-  - Example: "CY2021Q1" refers to January 2021 through March 2021.
-  - Example: "FY2022Q1" refers to July 2021 through September 2021.
+## Terminology
+- *Near Term*: These items are well-defined and actively in development.
+- *Mid Term*: Next to be picked up for development when capacity allows. The problem space & solution are known but we may still be defining implementation details.
+- *Long Term*: The team is investigating for the future but don’t have a clear solution in mind. This is the area most in flux as investigations can change feasibility & priority. Do not make plans reliant on these features as we cannot guarantee they will be implemented. 
 
-## Current Quarter
+## Timelines
 
-### CY2022Q1 / FY2022Q3
+### Near Term
+- Node token caching support with Redis 
+- Internal pipeline improvements for easier release cadence 
+- Deprecation of passport-azure-ad and replacement with a new lightweight token validator 
+- Updates required by the recent United State’s Executive Order on cybersecurity: Executive Order on Improving the Nation's Cybersecurity | The White House 
```

Comment:
```suggestion
- Updates required by the recent United States Executive Order on cybersecurity: Executive Order on Improving the Nation's Cybersecurity | The White House 
```

Reply:
Can you provide a link to the EO here as well?

Reply:
Yes that was my plan. Didn't realize the link didn't follow when I copied this out of Notebook I was working in. I will update.